source/languageHandler.py

```diff
@@ -105,13 +105,14 @@ def getAvailableLanguages(presentational=False):
 	displayNames = []
 	for entry in locales:
 		desc=getLanguageDescription(entry)
-		displayNames.append("%s, %s"%(desc,entry) if desc else entry)
+		displayNames.append(desc)
 	#Prepare a zipped view of language codes and descriptions.
 	# #7284: especially for sorting by description.
 	# Python 3: zip function changed from returning a list to an iterator, thus wrap this inside a list call.

```

Comment:
Feel free to remove this

Reply:
Ah, I'm sorry. I meant just the line about python 3. It is clear from the context that this should be a list. During the conversion to Python 3, it made sense to have these comments to see where a change came from, but now they are mostly obsolete.

Reply:
Yes, same issue for Italian such as in Spanish interface.

Reply:
OK. I have eventually made some clean-up and simplified this part of code. The Python 3 comment is gone.